specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json

```diff
@@ -193,7 +193,54 @@
                         }
                     }
                 }
-            }
+            },
+            "patch": {
+              "tags": [
+                  "HanaOnAzure"
+              ],
+              "operationId": "HanaInstances_UpdateTags",
```

Comment:
Today this API is only updating tags. Hence the operationId is relevant
​Tomorrow if you decide to expand the envelope and decide to support full blown patch, that is update any property of the HanaInstance then the operationId would have to be changed to "HanaInstances_Update" since it would then be updating tags and something more.
​
Changing the name of the operation at that time would be a breaking change for the sdks that get generated from the swagger spec. In swagger for a given path you can only have one verb (GET/PUT/PATCH/DELETE/etc). Just wanted to let you know about this.
​
- So the main question is: Are there any plans in the future to support a full blown patch?

@ravbhatnagar, @johanste - Is there any recommendation from ARM API review board or from ARM to name the operationId *_UpdateTags (if the API only supports patching tags) ?

Reply:
Good point! We decided to design for the future, and change the name to "HanaInstances_Update".